src/assets/js/connection.js

```diff
@@ -41,7 +41,8 @@ export default class Connection {
     if (self.inited) return;
     self.inited = true;
     this.unhide();
-    if (med.SFU_ENABLED) {
+    const isSFUEnabled = document.getElementById("mesh-toggle").checked ? !document.getElementById("mesh-toggle").checked : med.SFU_ENABLED;
```

Comment:
Don't know if this actually exists at the current time of execution - maybe it should depend on config flag here, and then later toggled in the UI stuff setting/unsetting checked on real time

Reply:
Just to clarify,

We will have in config.js a setting that we read out. That will be default, than we will switch in UI that will be passed as a parameter so it can be shared and read out for people joining. Does that make sense this way we can get rid of the const SFU_ENABLED

Reply:
Yes indeed

Reply:
- [x] Fixed it :)